website/pages/docs/integrate/nia-integration.mdx

```diff
@@ -17,7 +17,7 @@ Consul-Terraform-Sync executes one or more automation tasks with an appropriate
 
 [![NIA Architecture](/img/nia-highlevel-diagram.png)](/img/nia-highlevel-diagram.png)
 
--> Please note that the above indicated solution is a “push” based method and is not the only way to integrate network devices with Consul and drive Network Infrastructure Automation Integration. If your preferred method is to directly integrate with Consul without using Terraform, then please use [Consul Integration Program](/docs/integrate/partnerships).
+-> Please note that the above indicated solution is a “push” based method and is not the only way to integrate network devices with Consul and drive Network Infrastructure Automation Integration. If your preferred method is to directly integrate with Consul without using Terraform, then please use [Consul Integration Program](/docs/integrate/partnerships) or reach out to us directly at nia-integration-dev@hashicorp.com.
```

Comment:
Where does email sent to that address go?

Reply:
It goes to the Alliances team - Asvin & Dan are on it. I believe it might go to some other folks on the Alliances team as well.